ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue

```diff
@@ -232,6 +240,15 @@ export default defineComponent({
       invalidSpaces()
     )
 
+    const emailRules = customRules(
+      isEmail(),
```

Comment:
Email and Phone number are optional fields?

Reply:
Good Catch..  I added back the rule for required entry.